src/main/java/io/fabric8/elasticsearch/util/RequestUtils.java

```diff
@@ -73,6 +81,42 @@ public boolean isOperationsUser(RestRequest request) {
         }
         return allowed;
     }
+    
+    @SuppressWarnings("rawtypes")
+    public void assertUser(RestRequest request) throws Exception {
+        final String user = getUser(request);
+        final String token = getBearerToken(request);
+        ConfigBuilder builder = new ConfigBuilder().withOauthToken(token);
+        try (DefaultOpenShiftClient osClient = new DefaultOpenShiftClient(builder.build())) {
+            LOGGER.debug("Verifying user {} matches the given token.", user);
+            Request okRequest = new Request.Builder()
+                .addHeader(AUTHORIZATION_HEADER, "Bearer " + token)
+                .url(osClient.getMasterUrl() + "oapi/v1/users/~") 
+                .build();
+            String username = null;
+            try (Response response = osClient.getHttpClient().newCall(okRequest).execute()){
```

Comment:
Shouldn't this assignment be in the `try` block?

Reply:
This is generally true I believe.  Looking at the dependent code, however, this is a basically a builder pattern that is deferred until execute.  I'm going to assume the assignment is fine.